blog/_posts/2024-08-02-istio-with-gateway-api.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Using AKS-managed Istio External Ingress Gateway with Gateway API"
+description: "Learn how to use the Gateway API to manage the Istio external ingress gateway on AKS."
+date: 2024-08-06
+author: Paul Yu
+categories: networking, operations, add-ons
+---
+
+## Introduction
+
+Kubernetes is great at orchestrating containers, but it can be a bit tricky to manage traffic routing. There are many options and implementations that you, as a cluster operator has probably had to deal with. We have the default Services resource that can be used to expose applications within the cluster, but it is limited to routing based on layer 4 (TCP/UDP) and does not support more advanced traffic routing use cases. So enter the Ingress controller, which enabled layer 7 (HTTP) routing, and securing the North-South traffic using TLS, but it was not standardized and each vendor implementation learning a new set of resource annotations. When it came to managing and securing East-West traffic between services, we had to rely on a service mesh which is yet another layer of infrastructure to manage on top of Kubernetes. And we were in the same boat with configuration management with each vendor having their own custom resources ways of doing things.
```

Comment:
Minor nit, Maybe remove "So" and just have "Enter the Ingress..."

Reply:
Re-worded the paragraph.